app/views/albums/index.html.erb

```diff
@@ -0,0 +1,21 @@
+<h1>Albums</h1>
+<table>
+	<tbody>
+		<tr>
+			<th>Ranking</th>
+			<th>Name</th>
+			<th>Upvote</th>
+			<th>Downvote</th>
+		</tr>
+		<% @all_albums.each do |album|%>
+		<tr>
+			<td>Rank: <%=album.rank%></td>
+			<td><%=link_to album.name, album_path(album)%></td>
+			<td><%= button_to("Upvote", {action: "upvote", id: album.id}, {method: :patch, :class=> "success button"}) %></td>
```

Comment:
If we give a name to the upvote route in `config/routes.rb` we could use the route helper method here (e.g. `upvote_album_path(album.id)`) instead of the hash with `action` and `id`.